pureStorageDriver/templates/database/cockroach-operator.yaml

```diff
@@ -33,6 +33,10 @@ spec:
               value: pso-db-role
             - name: SECRET_NAMES
               value: pso-cockroach-node-certs,pso-cockroach-client-certs
+{{- if eq .Values.orchestrator.name "openshift"}}
```

Comment:
nit format: space before `}}`

Reply:
I dunno if we have to or not but just in case/for style, I'll add it. Thanks for the close reading!

Reply:
Fixed all the instances here and a few preexisting ones I happened to notice: we have issues like this around the entire chart though so I didn't go on a fixing rampage for now :joy: saving that for another day

Reply:
So uh
Turns out that the hyphen actually does have to have no spaces specifically. I'll be pushing another PR soon anyways so I'll fix it in there, but this currently won't install.